crypto/fipsmodule/aes/aes_test.cc

```diff
@@ -125,6 +125,73 @@ static void TestKeyWrap(FileTest *t) {
                                ciphertext.data(), ciphertext.size()));
 }
 
+static void TestEVPKeyWrap(FileTest *t) {
```

Comment:
is it valuable to add a version of `WycheproofKeyWrap` that uses this new API? Also are there any other interesting error cases that should be tested by themselves?